config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/openstack-k8s-operators/infra-operator
+  newName: quay.io/rabi/infra-operator
```

Comment:
Needs to be reverted

Reply:
Why is it even included? not in `.gitignore` I guess.